pkg/client/observatorium/observability_config.go

```diff
@@ -205,11 +282,13 @@ func (c *ObservabilityConfiguration) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&c.ObservabilityConfigRepo, "observability-config-repo", c.ObservabilityConfigRepo, "Repo for the observability operator configuration repo")
 	fs.StringVar(&c.ObservabilityConfigChannel, "observability-config-channel", c.ObservabilityConfigChannel, "Channel for the observability operator configuration repo")
-	fs.StringVar(&c.ObservabilityConfigAccessTokenFile, "observability-config-access-token-file", c.ObservabilityConfigAccessTokenFile, "File contains the access token to the observability operator configuration repo")
 	fs.StringVar(&c.ObservabilityConfigTag, "observability-config-tag", c.ObservabilityConfigTag, "Tag or branch to use inside the observability configuration repo")
 
 	fs.StringVar(&c.ObservabilityCloudWatchLoggingConfig.configFilePath, "observability-cloudwatchlogging-config-file-path", "secrets/observability-cloudwatchlogs-config.yaml", "Path to a file containing the configuration for Observability related to AWS CloudWatch Logging in YAML format. Only takes effect when --observability-cloudwatchlogging-enable is set")
 	fs.BoolVar(&c.ObservabilityCloudWatchLoggingConfig.CloudwatchLoggingEnabled, "observability-cloudwatchlogging-enable", false, "Enable Observability to deliver data plane logs to AWS CloudWatch")
+
+	fs.StringVar(&c.DataPlaneObservabilityConfig.configFilePath, "dataplane-observability-config", "secrets/dataplane-observability-config.yaml", "Path to a file containing the observability configuration for the data plane")
+	fs.BoolVar(&c.DataPlaneObservabilityConfig.Enabled, "dataplane-observability-config-enabled", true, "Determines if the data plane observability config file must be provided")
```

Comment:
Can we call it `--dataplane-observability-config-enable` instead?
Other flags use `*enable*` instead of `*enabled*`